packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1051,7 +1051,7 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   ///    calling [setupParentData] on the future parent node.
   ///  * The conventions for using the parent data depend on the layout protocol
   ///    used between the parent and child. For example, in box layout, the
-  ///    parent data is completely opaque but in sector layout the child is
+  ///    parent data is completely opaque but in sliver layout the child is
```

Comment:
Actually sector layout was intentional. It refers to:
https://github.com/flutter/flutter/blob/master/examples/layers/rendering/src/sector_layout.dart

Reply:
(In sliver layout, the child is not permitted to read fields either.)

Reply:
I'll close this PR then? seems' pretty pointless now ;)